espanol/dragon/devolverlo/devolverlo.md

```diff
@@ -0,0 +1 @@
+Con el temor de que el dragón se saliera de control y los atacara a ellos mismo, decidió tratar de devolverlo. Pero el vendedor no lo aceptaba de vuelta. Sin embargo, decidió ayudarlo a domesticarlo vendiendole...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.